src/OrchardCore/OrchardCore.DisplayManagement/TagHelpers/HtmlContentTagHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using System.Threading.Tasks;
+
+namespace OrchardCore.DisplayManagement.TagHelpers;
+
+[HtmlTargetElement("add-property", Attributes = "name", TagStructure = TagStructure.NormalOrSelfClosing)]
+public class HtmlContentTagHelper : TagHelper
```

Comment:
Rename Class and FileName to `AddPropertyTagHelper`

Reply:
Ok, renamed.